academy/launcher.py

```diff
@@ -43,7 +44,7 @@ def _run_agent_on_worker(agent: Agent[Any]) -> None:
 @dataclasses.dataclass
 class _ACB(Generic[BehaviorT]):
     # Agent Control Block
-    agent_id: AgentId[BehaviorT]
+    agent_info: RegistrationInfo[BehaviorT]
```

Comment:
Similar to with `Agent`, for ease of use let's keep an `agent_id` attribute and a separate `info`/`registration` attribute.